lnbits/core/static/js/wallet.js

```diff
@@ -795,6 +795,11 @@ new Vue({
         field: 'pending'
       })
       LNbits.utils.exportCSV(columns, this.payments)
+    },
+    pasteToTextArea: function () {
+      navigator.clipboard.readText().then(text => {
+        this.$refs.textArea.value = text
```

Comment:
is this protected against XSS?

Reply:
Yes
<img width="405" alt="image" src="https://github.com/lnbits/lnbits/assets/93376500/b26760f9-b833-4951-9787-cedb6c4e9369">